SimPEG/EM/FDEM/SrcFDEM.py

```diff
@@ -17,11 +16,6 @@ class BaseFDEMSrc(BaseEMSrc):
 
     freq = properties.Float("frequency of the source", min=0, required=True)
 
-    _ePrimary = None
-    _bPrimary = None
-    _hPrimary = None
-    _jPrimary = None
-
```

Comment:
stashing these in general is potentially dangerous - later, we should go through and stash on sources where the primary doesn't change.